assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,195 @@
+  }
+  margin-bottom: 24px;
+  text-align: start;
+  width: 100%;
+
+  a --country-selector--a-- {
+    color: $white;
+  }
+
+  a:hover,
+  button:hover {
+    text-decoration: var(--link--hover--text-decoration, underline);
+  }
+
+  button:focus {
+    border-radius: var(--link--focus--border-radius, 4px);
+    outline: var(--link--focus--border, 2px solid rgba(255, 255, 255, 0.4));
+    outline-offset: 0;
+  }
+
+  button:focus:not(:focus-visible) {
+    outline: 0;
+  }
+
+  .country-selector-toggle-container {
+    --country-selector--toggle-container-- {
+      border-bottom: 1px solid rgba(255, 255, 255, 0.5);
+    }
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-bottom: 24px;
+    transition: border 0s linear 0.5s;
+    vertical-align: middle;
+    width: 100%;
+  }
+
+  .control-toggle {
+    align-self: flex-start;
+    background: transparent;
+    border: none;
+    color: var(--site-footer--color, $white);
+    cursor: pointer;
+    text-align: start;
+    white-space: nowrap;
+    width: 100%;
+
+    &::before {
+      background-color: var(--site-footer--color, $white);
+      content: "";
+      display: inline-block;
+      height: 20px;
+      margin-inline-end: 8px;
+      mask: url("../../images/gp-g-logo.png") 0 0/16px 20px no-repeat;
+      vertical-align: middle;
+      width: 16px;
+    }
+
+    &::after {
+      background-color: var(--site-footer--color, $white);
+      content: "";
+      display: inline-block;
+      height: 12px;
+      margin-inline-start: 8px;
+      mask: url("../../images/chevron.svg") 0 0/16px 12px no-repeat;
+      transform: rotate(0.25turn);
+      transition: transform 300ms cubic-bezier(.86, 0, .07, 1);
+      width: 16px;
+    }
+  }
+
+  .countries-list {
+    --country-selector--list-- {
+      background-color: $active-blue;
+      font-weight: 500;
+    }
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.5s cubic-bezier(0, 1.04, .38, .37) 0s;
+  }
+
+  .countries-list .container {
+    padding: 24px 24px $space-lg 16px;
+
+    @include medium-and-up {
+      padding: 24px 0 $space-lg 0;
+    }
+  }
+
+  &.open {
+    .country-selector-toggle-container {
+      border-bottom: 1px solid rgba(255, 255, 255, 0);
+      transition-delay: 0s;
+    }
+
+    .control-toggle::after {
+      transform: rotate(-0.25turn);
+    }
+
+    .countries-list {
+      max-height: 5000px;
+      transition: max-height 1s linear 0s;
+    }
+  }
+
+  ul.countries {
+    column-count: 1;
+    list-style: none;
+    margin: 0;
+    padding-inline-start: $space-lg;
+
+    @include medium-and-up {
+      column-count: 3;
+    }
+
+    & li {
+      font-size: $font-size-sm;
+      line-height: 1;
+    }
+  }
+
+  li.international {
+    margin: 0;
+    padding: 0 0 $space-lg 0;
+  }
+
+  li.country-group-letter {
+    .initial {
+      --country-selector--initial-- {
+        color: rgba(255, 255, 255, 0.5);
+        font-weight: 700;
```

Comment:
The same here. If it's really necessary for editor, then add it, if not remove it from this block